frontend/src/components/ParkDetailsSeasonDates.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from "prop-types";
+import { useEffect, useMemo } from "react";
+import useDate from "@/hooks/useDate";
+import DateRange from "@/components/DateRange";
+
+function DateTypeRow({ dateType }) {
+  console.log("dateType", dateType);
+
+  return (
+    <tr>
+      <td>{dateType.dateType.name} dates</td>
+      <td>
+        <DateRange start={dateType.startDate} end={dateType.endDate} />
+      </td>
```

Comment:
A single dateType and season can have more than one date. 
<img width="1002" alt="Screenshot 2024-11-14 at 1 40 12 PM" src="https://github.com/user-attachments/assets/cd8a4e7d-2caf-47de-afa7-e6f9aae843ed">

Reply:
Woah, I definitely didn't finish this file 😆 I'll clean it up.